runtime/src/lib.rs

```diff
@@ -320,11 +293,9 @@ impl pallet_balances::Config for Runtime {
     type Event = Event;
     type ExistentialDeposit = ExistentialDeposit;
     type MaxLocks = MaxLocks;
+    type MaxReserves = MaxReserves;
     type WeightInfo = ();
-}
-
-parameter_types! {
-    pub const TransactionByteFee: Balance = 1;
+    type ReserveIdentifier = [u8; 8];
 }
 
 impl pallet_transaction_payment::Config for Runtime {
```

Comment:
why isn't the value of `OnChargeTransaction` the same as in standalone `type OnChargeTransaction = CurrencyAdapter<Balances, DealWithFees>;` https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR549

Reply:
why isn't `FeeMultiplierUpdate`'s value the same as in standalone chain https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR552

Reply:
why not call `GeneralCouncilInstance` `CouncilCollective` like in standalone chain, and give its `WeightInfo` a value of `pallet_collective::weights::SubstrateWeight<Runtime>` like in standalone chain?

Reply:
why isn't `GeneralCouncilMembershipInstance` like `EnsureRootOrHalfCouncil` in our standalone chain https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR646 with similar values?

Reply:
we're not using `GeneralCouncilProvider` in the standalone chain code, do we need to add it ?
in the standalone chain we use pallet_tips with `type Tippers = Elections;`

Reply:
I don't know. @festelo Can you answer this? I haven't changed this.

Reply:
it looks like this will just have to be addressed in issue https://github.com/DataHighway-DHX/node/issues/227